plugins/client-resources/src/connection.ts

```diff
@@ -30,22 +30,31 @@ class DeferredPromise {
   }
 }
 
-class Connection implements Storage {
-  private websocket: WebSocket | null = null
+class Connection implements ClientConnection {
+  private websocket: ClientSocket | null = null
   private readonly requests = new Map<ReqId, DeferredPromise>()
   private lastId = 0
+  private readonly interval: number
 
   constructor (private readonly url: string, private readonly handler: TxHander) {
     console.log('connection created')
-    setInterval(() => {
+    this.interval = setInterval(() => {
       console.log('ping')
```

Comment:
Could we get rid of this logging within PR please? Just to prevent flooding node services logs.

Reply:
logging removed